src/Space.ts

```diff
@@ -87,6 +87,7 @@ class Space extends EventEmitter<{ membersUpdate: SpaceMember[] }> {
 
     member.isConnected = message.action !== 'leave';
     member.profileData = message.data?.profileData ?? member.profileData;
+    member.location = member.location ? member.location : message.data?.location ?? null;
```

Comment:
Shouldn't location be `currentLocation`?

Reply:
At this point the `location` is on the Presence message rather than the location change so it should always be `message.data.location`